mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClustersManagerActor.java

```diff
@@ -59,7 +61,8 @@ class ResourceClustersManagerActor extends AbstractActor {
     private final RpcService rpcService;
     private final MantisJobStore mantisJobStore;
 
-    private final Map<ClusterID, ActorRef> resourceClusterActorMap;
+    // Cluster Id to <ResourceClusterActor, ResourceClusterScalerActor> map.
+    private final Map<ClusterID, Pair<ActorRef, ActorRef>> resourceClusterActorMap;
```

Comment:
Can we use something else apart from Pair? From my experience, Pairs always evolve into a higher cardinality data structure.